source/tools/log_handler.py

```diff
@@ -0,0 +1,264 @@
+# Copyright (C) 2016 iNuron NV
+#
+# This file is part of Open vStorage Open Source Edition (OSE),
+# as available from
+#
+#      http://www.openvstorage.org and
+#      http://www.openvstorage.com.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+# as published by the Free Software Foundation, in version 3 as it comes
+# in the LICENSE.txt file of the Open vStorage OSE distribution.
+#
+# Open vStorage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY of any kind.
+
+"""
+Contains the loghandler module
+"""
+
+import os
+import sys
+import time
+import inspect
+import socket
+import logging
+import itertools
+
+
+class OVSFormatter(logging.Formatter):
+    def formatTime(self, record, datefmt=None):
```

Comment:
It inherits from a python class (```logging.Formatter```) and overrides a build-in method, and apparently they sometimes violate their own rules.